### PR TITLE
Mobile changes

### PR DESCRIPTION
**Pull request by JRascagneres**
Changes to support mobile 

Specifications here:
https://docs.google.com/document/d/1WmdvNU2Fb9d4e9xBU-k17URd8OvL1K1EZnqGV_I9F48/edit?usp=sharing

Works with https://github.com/couchbaselabs/greenboard/pull/4

Requires removing mobile and creating cblite and sync_gateway buckets. As discussed with @korry8911 this would take us to 11 buckets so ideally would need to remove any existing ones that aren't required.

No changes were made to whatever jinja-manual is so if necessary we will need to change that too. Whats the purpose of this?

### CODE REVIEW

File: jinja.py [hunk 1]
```diff
@@ -17,13 +17,13 @@
 UBER_PASS = os.environ.get('UBER_PASS') or ""
 
 JOBS = {}
-HOST = '172.23.98.63'
+HOST = '127.0.0.1'
```
Comment (korry8911) [inline jinja.py]: I think this needs to be changed back to original IP

File: jinja.py [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
 
 if len(sys.argv) == 2:
     HOST = sys.argv[1]
 
 
-def getJS(url, params=None, retry=5, append_api_json=True):
+def getJS(url, params=None, retry=10, append_api_json=True):
```
Comment (korry8911) [inline jinja.py]: Why raise it to 10?

File: jinja.py [hunk 3]
```diff
@@ -422,6 +422,7 @@ def storeTest(jobDoc, view, first_pass=True, lastTotalCount=-1, claimedBuilds=No
                 pass  # ok, this is new doc
 
             try:
+                print(key)
```
Comment (korry8911) [inline jinja.py]: should we be printing this?

File: jinja.py [hunk 4]
```diff
@@ -592,11 +593,23 @@ def pollTest(view):
             continue
 
         for job in j["jobs"]:
+            print(job["name"])
```
Comment (korry8911) [inline jinja.py]: same here... why print this?

File: jinja.py (hunk 1 above)
Comment (JRascagneres) [reply jinja.py]: Ah yes! Thanks for catching!

File: jinja.py (hunk 2 above)
Comment (JRascagneres) [reply jinja.py]: Ah yes, also local testing. Had a lot of retries so bumped it. Will put it back

File: jinja.py (hunk 3 above)
Comment (JRascagneres) [reply jinja.py]: Debugging. Will revert.

File: jinja.py (hunk 4 above)
Comment (JRascagneres) [reply jinja.py]: Debugging. Will revert.

**Review by korry8911 — APPROVED** [review]
looks good to me

**Pull request merged**